test/unit/services/DeclarationServiceSpec.scala

```diff
@@ -79,43 +79,43 @@ class DeclarationServiceSpec extends UnitSpec with MockitoSugar {
 
       "send transformed xml to connector" in new SetUp() {
 
-        val result: Either[Result, Unit] = send()
+        val result: Either[Result, Option[NrSubmissionId]] = send()
 
-        result shouldBe Right(())
+        result shouldBe Right(None)
```

Comment:
If this is always None.... why is it an Option? If it is Never Set then why is it returned as None? If it can be set or is set then do we need to test when it is set?

Reply:
Its is always none just in this test - StandardDeclarationSubmissionServiceSpec tests the cases when this returns a value.